tcms/testruns/views.py

```diff
@@ -861,8 +851,6 @@ def get_context_data(self, **kwargs):
             'test_case_run_bugs': bug_ids,
             'mode_stats': mode_stats,
             'summary_stats': summary_stats,
-            'jira_url': jira_url,
-            'bugzilla_url': bugzilla_url
```

Comment:
See my above comment, opening bugs or issues should not be removed in this commit.